macros/edr/materializations/test/test.sql

```diff
@@ -170,6 +171,8 @@
   {% set test_results_description %}
       {% if has_anomaly_score %}
           {{ elementary.insensitive_get_dict_value(latest_row, 'anomaly_description') }}
+      {% elif rows_with_score %}
```

Comment:
Maybe it makes sense to remove `has_anomaly_score` and keep only `rows_with_score`? (and also apply to the log above)